src/components/artwork/index.tsx

```diff
@@ -1,42 +1,70 @@
 import * as React from "react"
-import * as Relay from "react-relay"
+
 import styled from "styled-components"
-import ArtworkMetadata from "./metadata"
+import ArtworkMetadata, { ArtworkMetadataProps } from "./metadata"
+import createContainer, { RelayProps } from "./relay"
+
+const ImageContainer = styled.div`
+  height: ${props => `${props.size}px`};
+  width: ${props => `${props.size}px`};
```

Comment:
:| this breaks the `ArtworkGrid`, I think we are going to have to make separate containers for non-grid artworks.